README.md

```diff
@@ -7,20 +7,44 @@ The API spec all the proof-of-concepts: https://app.swaggerhub.com/apis-docs/bil
 Crowdsourced brainstorm of problems we want to solve: https://pad.riseup.net/p/BecKdThFsevRmmG_tqFa-keep
 
 # Setup
-- Make sure you have Postgres 11.1 installed
-- Create a Postgres user by typing `createuser --interactive --pwprompt`
-- Create a database in Postgres called cbv3_django_prototype by typing `createdb cbv3_django_prototype`
-- Type `pip install -r requirements.txt` in your command line
+
+- Fork this repo
+- Make sure you have Python 3 and Postgres 11.1 installed
+
+---
+
+You can either set up Postgres locally or using Docker.
+
+### Set up Postgres with Docker:
+
+[https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198](https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198)
```

Comment:
Perhaps we can distill this article to the vital steps, and include them in the README?
```suggestion
[Docker](https://www.docker.com/) makes it easy to [install and manage Postgres](https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198).

With Docker installed, try the following steps:

1. `docker pull postgres`
2. `mkdir -p $HOME/docker/volumes/postgres`
3. docker run --rm   --name pg-docker -e POSTGRES_PASSWORD=docker -d -p 5432:5432 -v $HOME/docker/volumes/postgres:/var/lib/postgresql/data  postgres
```

Postgres should then be accessible via localhost, e.g. from Django or with the `psql` command line tool:

```
psql -h localhost -U postgres -d postgres
```

Reply:
Thank you! And sorry, I was being a bit lazy/doing a rush job of this PR last night. :P